ci/docker/Dockerfile.build.armv6

```diff
@@ -30,17 +30,24 @@ FROM dockcross/linux-armv6
 # extract ccache binary into latest context
 COPY --from=ccachebuilder /usr/local/bin/ccache /usr/local/bin/ccache
 
+RUN apt-get update
+RUN apt-get install -y unzip
+
 ENV ARCH armv6l
+ENV FC=/usr/bin/${CROSS_TRIPLE}-gfortran
```

Comment:
why?

Reply:
Because the compilers are set in the base [dockcross](https://github.com/dockcross/dockcross) image, but previously fortran compiler was not set. I've [added it](https://github.com/dockcross/dockcross/pull/228).

Reply:
ok